src/peripheral/mod.rs

```diff
@@ -443,9 +443,9 @@ pub enum FpuAccessMode {
     Privileged,
 }
 
-const SCB_CPACR_FPU_MASK: u32 = 0x00780000;
-const SCB_CPACR_FPU_ENABLE: u32 = 0x00280000;
-const SCB_CPACR_FPU_USER: u32 = 0x00500000;
+const SCB_CPACR_FPU_MASK: u32 = 0x00f00000;
```

Comment:
Could you rewrite these as  `0b1111 << 20`? I think that format would increase readability and avoid off by one errors.